irohad/main/config/impl/gflags_config.cpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "gflags_config.hpp"
+#include <sstream>
+#include "main/config/flags.hpp"
+
+namespace iroha {
+  namespace config {
+
+    GFlagsConfig::GFlagsConfig() { load(); }
+
+    void GFlagsConfig::load() {
```

Comment:
What's the point of moving code of constructor to `load`? `FLAGS_*` can only set once, I guess, and so the function should be called once

Reply:
if you change config from your program (change env var, for example), then you might want to reload your env var.

Reply:
Also, you can not overload constructor. 
The purpose of  this class is to provide access to flags.
they way how it exactly executes `load()` should be  hidden in implementation class